pallets/social/src/lib.rs

```diff
@@ -38,7 +35,7 @@ pub struct Blog<T: Trait> {
 #[derive(Encode, Decode, Clone, Eq, PartialEq, RuntimeDebug)]
 pub struct BlogUpdate<AccountId> {
   pub writers: Option<Vec<AccountId>>,
-  pub slug: Option<Vec<u8>>,
+  pub handle: Option<Vec<u8>>,
```

Comment:
Hande should be optional.
```suggestion
  pub handle: Option<Option<Vec<u8>>>,
```